ContractsApi/V1/Boundary/Requests/CreateContractRequestObject.cs

```diff
@@ -22,6 +22,7 @@ public class CreateContractRequestObject
         public string Brma { get; set; }
         public bool? IsActive { get; set; }
         public bool IsApproved { get; set; }
```

Comment:
The `IsApproved` flag is no longer required, as it is now covered by the new status,

Reply:
Yeah, I just left it there so as not to break the FE until we introduce the new one. but now I'm second guessing this...would it actually break? 😅

Reply:
I don't think so, because I believe the FE currently handles the absence of `isApproved`, but worth checking.

Reply:
Due to potential issues with the PATCH call and to err on the side of caution, we will keep the `isApproved` flag for now.